samples/drivers/spi_controller/README.rst

```diff
@@ -0,0 +1,21 @@
+.. zephyr:code-sample:: spi-controller
```

Comment:
Ah here are your changes applying the comments we made. That's not how it works: you need to fixup your original commits. (and a few comments I made are still valid)